x/wbank/keeper/msg_server.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/hashicorp/go-metrics"
 )
 
+// MsgServer implements the bank Msg service.
+type MsgServer interface {
```

Comment:
correct me if I am wrong, but seems like changes to this file are redundant and you can revert them.